src/gui/macutilities.mm

```diff
@@ -81,3 +82,15 @@ void displayNotification(const QString &title, const QString &message)
         [[NSUserNotificationCenter defaultUserNotificationCenter] deliverNotification:notification];
     }
 }
+
+void openFiles(QSet<QString> &pathsList)
```

Comment:
`const QSet<QString> &pathsList`

Reply:
Another thing, maybe it's better to wrap all functions in this file in `MacUtils` namespace?
Of course I'm not requesting you to move it all now. For now, just this newly added function.

Reply:
I was thinking about the same thing, but the point indeed is to move them all.
Since there are only 4 functions, will you be fine if I move them all together within this pr?

Reply:
>Since there are only 4 functions, will you be fine if I move them all together within this pr?

That would be great! and please do it in a separate commit.